packages/remix-forms/src/formAction.server.ts

```diff
@@ -64,7 +64,7 @@ export async function performMutation<
                 (error) => error.message,
               )
             : undefined,
-      },
+      } as FormErrors<Schema>,
```

Comment:
This was needed probably due to [this change](https://github.com/SeasonedSoftware/remix-domains/pull/31) in the [`errorMessagesForSchema`](https://github.com/SeasonedSoftware/remix-domains/#errormessagesforschema) in [remix-domains](https://github.com/SeasonedSoftware/remix-domains/)